### DISCUSSION TITLE
Compressions compressions compression!

### DISCUSSION THREAD

**Discussion by drickles** [rodit/SnapMod#159] [General]
Per what I read online snapchat has a 1gb video size limit and 5mb image size limit when it came to media upload limits but what I've noticed on my device is everything, absolutely everything I try to upload gets compressed. Doesnt matter if its a 15kb image or a 2mb image. Doesnt matter if its a 25mb video or a 200mb video. Whether I send the media as a snap or "posted from gallery" onto my story. On my phone's story board it looks great but when I check it out from other accounts on other devices there's a clear compression and loss of quality to the media and I don't know how to fix it. I just want to know if this happens to anyone else and if so is there a way to fix it. I use fast Internet and WiFi so I'm not limited by slow Internet or cellular data.

**Comment by rodit** [comment]
I'll look into this, I'm quite familiar with Snapchats upload process by now so I'll see if I can find the compression code and skip it/reduce it.

**Reply by drickles** [reply]
Thanks a lot. It doesn't even make sense. Not everything needs to be compressed so aggressively.

**Comment by rodit** [comment]
You can use the 'Force Source Encoding' option to disable Snapchat compression now.

**Reply by drickles** [reply]
And it works perfectly.. thanks a lot for this.
